bin/verilator_includer

```diff
@@ -11,9 +11,15 @@
 require 5.005;
 use warnings;
 print "// DESCR" . "IPTION: Generated by verilator_includer via makefile\n";
+print "#pragma once\n";
 foreach my $param (@ARGV) {
     if ($param =~ /^-D([^=]+)=(.*)/) {
         print "#define $1 $2\n";
+    } elsif (-d $param) {
+        my @cpps = glob("$param/*cpp");
```

Comment:
Maybe instead pass in the glob directly with quoted asterisk, as then it's more obvious what is going on.